src/components/analyses/ids.js

```diff
@@ -0,0 +1,91 @@
+export default {
+    CHECKBOX: ".checkbox",
```

Comment:
Just picking a random spot - these ids don't need the `.` as long as you're using `build` for your ids.